### ISSUE TITLE
Is there any way to see the class coverage over xml report

### ISSUE THREAD

**Issue by alinaeftn** [Kotlin/kotlinx-kover#271]
Hi!

I am interested in looking to see if there is a way to display the coverage at a file level in an HTML representation from the XML report.

Right now, we are using Jacoco and Jenkins Jacoco plugin to display the coverage on each PR. This means that we can break down to classes and see an html representation of what lines are covered in a file and what are not.

Trying Kover, I found out that we can no longer see this on the XML report. The files cannot be opened to see the actual lines that are covered by tests and the ones that are not.

Is there a way to configure Kover to include an html representation in the XML report? Or can we achieve this somehow?

Also, can we integrate this with Sonarqube? Right now we are facing the same problem with lines not being displayed for coverage in Sonarqube.

Thank you!

**Comment by shanshin** [comment]
Hi,
may you clarify, what kind of HTML report do you mean, which is generated by the Kover by the `koverHtmlReport` command, or some of the report from the `Jenkins Jacoco plugin`?

**Comment by alinaeftn** [comment]
@shanshin , I am referring to the HTML from Kover. Let me elaborate a little bit more.

I am using the XML report for reporting coverage from Jenkins. Meaning, I am running the `koverMergedReport` task and use the jacocoAdapter to display the coverage.
```
sh './gradlew clean build koverMergedReport --parallel'
sh 'cp build/reports/kover/merged/xml/report.xml build/koverTestReport.xml'
publishCoverage adapters: [jacocoAdapter('build/reports/kover/merged/xml/report.xml')]
```

 But when I got in Jenkins to Coverage Report -> File Coverage I cannot click on a file to see what are the lines that are covered and what are not. I cannot have a visual input about what is tested. 

Is there a way to do this?

**Comment by alinaeftn** [comment]
@shanshin , the reports were generated with `koverMergedReport` and `jacocoRootReport`. No change was applied to the tasks, only xml and html enablement. 

So I am assuming the default settings for this are not showing the same output or maybe not in the format in which Jenkins can interpret them and show the covered lines.

**Comment by shanshin** [comment]
Closed as duplicate of #272
